diff -urN kcmsystemd-1.1.0/CMakeLists.txt kcmsystemd-1.1.0-patched/CMakeLists.txt
--- kcmsystemd-1.1.0/CMakeLists.txt	2015-06-10 15:33:29.130556823 +1000
+++ kcmsystemd-1.1.0-patched/CMakeLists.txt	2015-06-10 15:33:48.317557696 +1000
@@ -25,12 +25,12 @@
              I18n
              Auth)
 
-message(STATUS "Searching for libsystemd...")
-find_library(SYSTEMD_FOUND NAMES systemd)
+message(STATUS "Searching for libsystemd-journal...")
+find_library(SYSTEMD_FOUND NAMES systemd-journal)
 if(SYSTEMD_FOUND)
   message(STATUS "Found at: " ${SYSTEMD_FOUND})
 else(SYSTEMD_FOUND)
-  message(FATAL_ERROR "Unable to find libsystemd. Configuration failed!")
+  message(FATAL_ERROR "Unable to find libsystemd-journal. Configuration failed!")
 endif(SYSTEMD_FOUND)
 
 include(KDEInstallDirs)
diff -urN kcmsystemd-1.1.0/src/CMakeLists.txt kcmsystemd-1.1.0-patched/src/CMakeLists.txt
--- kcmsystemd-1.1.0/src/CMakeLists.txt	2015-06-10 15:33:29.129556822 +1000
+++ kcmsystemd-1.1.0-patched/src/CMakeLists.txt	2015-06-10 15:33:57.360558107 +1000
@@ -14,7 +14,7 @@
                       KF5::I18n
                       KF5::ConfigWidgets
                       ${Boost_LIBRARIES}
-                      -lsystemd)
+                      -lsystemd-journal)
 install(TARGETS kcm_systemd DESTINATION ${CMAKE_INSTALL_PLUGINDIR})
 
 qt5_use_modules(kcm_systemd Widgets DBus)
