diff -urN kcmsystemd-1.1.0/src/kcmsystemd.cpp kcmsystemd-1.1.0-patched/src/kcmsystemd.cpp
--- kcmsystemd-1.1.0/src/kcmsystemd.cpp	2015-03-03 22:08:36.000000000 +1000
+++ kcmsystemd-1.1.0-patched/src/kcmsystemd.cpp	2015-06-10 15:14:29.951504982 +1000
@@ -1638,8 +1638,8 @@
   if (a == edit)
   {
     QProcess kwrite (this);
-    QString alt1 = QString(kdePrefix + "/lib/libexec/kdesu");
-    QString alt2 = QString(kdePrefix + "/bin/kdesu");
+    QString alt1 = QString("/usr/lib64/libexec/kf5/kdesu");
+    QString alt2 = QString("/usr/lib/libexec/kf5/kdesu");
     if (QFile(alt1).exists())
       kwrite.startDetached(alt1, QStringList() << "-t" << "--" << "kwrite" << frpath);
     else if (QFile(alt2).exists())
