diff -Naur systemd-kcm-1.2.0/src/CMakeLists.txt systemd-kcm-1.2.0.tpg/src/CMakeLists.txt
--- systemd-kcm-1.2.0/src/CMakeLists.txt	2015-07-02 19:41:44.000000000 +0000
+++ systemd-kcm-1.2.0.tpg/src/CMakeLists.txt	2015-07-06 18:49:46.322646764 +0000
@@ -17,7 +17,8 @@
                       KF5::ConfigWidgets
                       KF5::Service
                       ${Boost_LIBRARIES}
-                      -lsystemd)
+                      -lsystemd
+                      -lsystemd-journal)
 install(TARGETS kcm_systemd DESTINATION ${CMAKE_INSTALL_PLUGINDIR})
 
 qt5_use_modules(kcm_systemd Widgets DBus)
